web-app/HeimScripts/heatmap/summary.R

```diff
@@ -151,12 +257,17 @@ extract_measurements <- function(datasets)
 produce_summary_stats <- function(measurement_tables, phase)
 {
   # construct data.frame to store the results from the summary statistics in
-  result_table <- as.data.frame(matrix(NA, length(measurement_tables),12, 
+  result_table <- as.data.frame(matrix(NA, length(measurement_tables),13, 
```

Comment:
(nitpicking)
Wouldn't it be cleaner to:
-  assign 13 to some named constant or put a comment explaining why 13
-  explicitly use nrow and ncol in matrix() call